packages/nimble-components/src/wafer-map/modules/matrix-renderer.ts

```diff
@@ -0,0 +1,55 @@
+import type { WaferMap } from '..';
+import { HoverDieOpacity } from '../types';
+
+/**
+ * Responsible for drawing the dies inside the wafer map, adding dieText and scaling the canvas
+ */
+export class MatrixRenderer {
```

Comment:
What's the idea behind the term "Matrix" here? Nothing on the API surface seems directly related to a matrix.